CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+## v0.7.0-beta (09/09/2019)
+
+**Common**
+
+* Introduced new abstract `olp::thread::TaskScheduler` to be used for async context; provided default thread pool implementation.
+* Added `UserAgent` HTTP header to mark all triggered requests.
+* Added China Lookup API URLs.
+* Introduced new HTTP abstraction layer `olp::http::Network` without singleton usage. Platform dependent implementations adapted and aligned with coding style.
+* `olp::network::Network` with according implementations and helper classes removed.
+* Changed `olp::client::CancellationContext` to shared pimpl so we can remove the `std::shared_ptr` wrapper when using it across multiple requests.
+* Improved logging in core components.
+
+**olp-edge-sdk-cpp-authentication**
+
+* **Breaking Change** - Use new HTTP layer as input parameter for all requests. This is a mandatory parameter and users must provide it.
+* Use new `olp::thread::TaskScheduler`; provided as input parameter.
+* Removed usage of raw `std::thread` and switched to `olp::thread::TaskScheduler` for callbacks. If not provided callbacks are executed synchronously.
```

Comment:
If unprovided callbacks executed synchronously.

Reply:
```suggestion
* Removed usage of raw `std::thread` and switched to `olp::thread::TaskScheduler` for callbacks. If unprovided, callbacks are executed synchronously.
```

To mark that unprovided belongs to TaskScheduler and not callbacks.